types/emoji-mart/emoji-mart-tests.tsx

```diff
@@ -1,6 +1,6 @@
-// Port of https://github.com/missive/emoji-mart/blob/master/src/components/emoji.js
+// Port of https://missive.github.io/emoji-mart/
```

Comment:
This should link to the source code of the page; looks like it's been moved to https://github.com/missive/emoji-mart/blob/v2.8.0/stories/index.js